src/Apps/Order2/Routes/Checkout/Components/Order2CheckoutLoadingSkeleton.tsx

```diff
@@ -48,12 +48,13 @@ const Order2CollapsibleOrderSummarySkeleton: React.FC<
   Order2CollapsibleOrderSummarySkeletonProps
 > = props => {
   return (
-    <GridColumns pt={[0, "50px"]} px={[0, 0, 4]}>
-      <Column span={[12, 12, 7]} start={[1, 1, 1]}>
+    <GridColumns py={[0, 4]} px={[0, 0, 4]}>
+      <Column span={[12, 12, 7]} width="100%">
         <Box
-          maxWidth={"754px"}
           width="100%"
-          justifySelf={["center", "center", "end"]}
+          maxWidth="754px"
+          mx={["auto", "auto", 0]}
```

Comment:
#super minor but for clarity - should this be `mr` as `ml` right after will going to overwrite margin left anyways leaving this controlling only right margin?